src/fields/jsgrid.field.select.js

```diff
@@ -107,10 +107,10 @@
                     .text(text)
                     .appendTo($result);
 
-                $option.prop("selected", (selectedIndex === index));
             });
 
             $result.prop("disabled", !!this.readOnly);
+			$result.prop('selectedIndex', selectedIndex);
```

Comment:
Would you mind to do two small cosmetic changes:
 - put spaces instead of tabs to align lines
 - put double quotes instead of single for consistency